applovin_max/ios/Classes/AppLovinMAX.m

```diff
@@ -152,8 +167,34 @@ - (void)initializeWithPluginVersion:(NSString *)pluginVersion sdkKey:(NSString *
         }
     }
     
+    ALSdkSettings *settings = [[ALSdkSettings alloc] init];
```

Comment:
We should also update the other settings that we update below `testDeviceIdentifiersToSet`, `verboseLoggingToSet`, `creativeDebuggerEnabledToSet` and `locationCollectionEnabledToSet` here. 

That way, the settings are passed in before SDK init.